gulp/copy-assets.js

```diff
@@ -0,0 +1,18 @@
+/*
+  copy.js
+  ===========
+  copies images and javascript folders to public
+*/
+
+var gulp = require('gulp')
+var config = require('./config.json')
+
+gulp.task('copy-assets', function () {
+  return gulp.src(['!' + config.paths.assets + 'sass{,/**/*}', config.paths.assets + '/**'])
```

Comment:
I would put a newline after the comma here so it doesn't look like one big rambling path, but feel free to ignore my nit.